src/main/kotlin/com/team4099/robot2023/commands/elevator/ElevatorCharacterizeCommand.kt

```diff
@@ -0,0 +1,43 @@
+package com.team4099.robot2023.commands.elevator
+
+import com.team4099.robot2023.config.constants.ElevatorConstants
+import com.team4099.robot2023.subsystems.elevator.Elevator
+import edu.wpi.first.wpilibj.RobotBase
+import edu.wpi.first.wpilibj.simulation.RoboRioSim
+import edu.wpi.first.wpilibj2.command.CommandBase
+import org.littletonrobotics.junction.inputs.LoggedPowerDistribution
+import org.team4099.lib.units.base.meters
+import org.team4099.lib.units.derived.inVolts
+import org.team4099.lib.units.derived.volts
+import org.team4099.lib.units.perSecond
+
+class ElevatorCharacterizeCommand(val elevator: Elevator) : CommandBase() {
```

Comment:
this isn't really characterization this is just getting your kS + kG value: the amount of voltage needed to overcome static friction and gravity